### PR TITLE
Add GPU support section in WSL 2 docks

### PR DESCRIPTION
**Pull request by simonferquel**
Signed-off-by: Simon Ferquel <simon.ferquel@docker.com>

### Unreleased project version (optional)

This is related to Docker Desktop 3.1.0

### CODE REVIEW

**Review by usha-mandya — APPROVED** [review]
LGTM

File: docker-for-windows/wsl.md [hunk 1]
```diff
@@ -122,7 +123,9 @@ Run "nbody -benchmark [-numbodies=<numBodies>]" to measure performance.
         -cpu              (run n-body simulation on the CPU)
         -tipsy=<file.bin> (load a tipsy model file for simulation)
 
-NOTE: The CUDA Samples are not meant for performance measurements. Results may vary when GPU Boost is enabled.
+>**Note**
```
Comment (simonferquel) [inline docker-for-windows/wsl.md:126]: This is actually the output of the docker command, it should not be re-styled (I am going to change it back)

Comment (usha-mandya) [reply docker-for-windows/wsl.md]: Aah OK. Makes sense. Thanks

Comment (usha-mandya) [reply docker-for-windows/wsl.md]: Could you add the backticks (```) after the Note in that case?

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**